install/install.md

```diff
@@ -59,6 +59,11 @@ If the server has not started or needs to be restarted / stop, use the included
 <br/>
 
 ### **pip Issues**
+<i class="fas fa-info-circle"></i> The CUDA version on your system must match the pip CUDA version you install (`-cu11` or `-cu12`). If you're unsure of your system CUDA version, the code below automatically determines it:<br/>
```

Comment:
Can we verify this? This will install packages that match the driver and not the toolkit, if I understand the behavior of `nvidia-smi` correctly. I believe if you install CUDA 12 drivers (e.g. 525) and a CUDA 11 toolkit, this will install an incompatible version of RAPIDS (it'll install `-cu12` but you need to match the toolkit, `-cu11`). You can test this on some of the ASE lab machines, which have driver 525 and CUDA 11.8 toolkit installed.

Reply:
Yeah... I wouldn't rely on CUDA being installed to `/usr/local`. Can we just say "install the version that matches your CUDA toolkit version" and not attempt to write user-proof code to do it automatically?

Reply:
(For a counter-case of the `/usr/local` code, you could install the CUDA toolkit via conda and not install a system CTK, only a driver.)

Reply:
Yup, agree we should just leave it to the user to match them. If we get lots of issues filed due to challenges here we can reconsider.